### DISCUSSION TITLE
What are the best config options for managing media that spans multiple mounts?

### DISCUSSION THREAD

**Discussion by jordanlambrecht** [cross-seed/cross-seed#803] [Q&A]
I have multiple hard drives for different media types. This is my current mount list:  

/drive01/data/torrents/_complete/tv
/drive02/data/torrents/_complete/movies
/drive03/data/torrents/_complete/anime

I would like to use hard links so I can delete torrent files individually once they reach their ratio/time max. Right now, my config looks something like this:

```javascript
  flatLinking: true,
  action: "inject",
  duplicateCategories: true,
```

But, if I delete one torrent it deletes all of them, which is kind of a pain:

![Uploading Screenshot 2024-09-27 at 11.28.s@2x.png…]()

Does anyone have a better solution?

side note: I'm using qbitmanage in conjunction with cross-seed, but Force ATM is disabled.

**Comment by ShanaryS** [comment]
`flatLinking: false`?

**Reply by jordanlambrecht** [reply]
According to the docs, "This option will otherwise link any matches to a tracker-specific folder inside of linkDir (if set)". But I can't use linkDir because I have multiple mounts.

I would have to set `linkDir: /drive01/data/torrents/links` or something, which would duplicate files that are on other mounts, i.e `/drive02/data/torrents/_complete/movies`

Or am I mistaken?

**Reply by mmgoodnow** [reply]
You can use symlinks

**Reply by ShanaryS** [reply]
A drive pool with hardlinks should also work. The hardlinks will always stay on the source drive, but all the paths in qbit will be through the pooled drive.

**Reply by jordanlambrecht** [reply]
@mmgoodnow I thought about that, but the problem would remain where if I delete the source file, it would break all of the cross-seeds

@ShanaryS Sadly, not an option =/ I have a Qnap DAS, a Synology NAS, and a physical external hdd

**Reply by mmgoodnow** [reply]
well, those are your options. you'll have to choose one
